packages/bpk-component-dialog/src/BpkDialogInner.tsx

```diff
@@ -30,6 +30,7 @@ const getClassName = cssModules(STYLES);
 const BpkDialogInner = (props: Props) => {
   const {
     ariaLabel,
+    ariaModal = true,
```

Comment:
You will need to make sure you add this to the `BpkDialogInner.d.ts` file so that it will come through in the TS types otherwise it will cause issues

Reply:
Where exactly @olliecurtis? I am not sure where to add `ariaModal` to the aforementioned file 😕.